template/README.md.js

```diff
@@ -0,0 +1,16 @@
+import { File } from '@asyncapi/generator-react-sdk';
+export default function readmeFile({ asyncapi }) {
+  return <File name={'README.md'}>
+    {
+      `
+    # ${asyncapi.info().title() }
```

Comment:
You should create something like 
```js
# ${asyncapi.info().title()}

${asyncapi.info().description()}
        
Available nats encodings:
https://nodejs.org/api/buffer.html#buffer_buffers_and_character_encodings
```

in your code, everything will have an indentation and it doesn't look good in md files :)